pulp_container/app/settings.py

```diff
@@ -1,4 +1,11 @@
+from django.conf import settings
+
 DRF_ACCESS_POLICY = {
     "dynaconf_merge_unique": True,
     "reusable_conditions": ["pulp_container.app.global_access_conditions"],
 }
+
+REST_FRAMEWORK = settings.REST_FRAMEWORK
+REST_FRAMEWORK.update(
+    {"EXCEPTION_HANDLER": "pulp_container.app.exceptions.unauthorized_exception_handler"}
```

Comment:
Won't this set the exception handler for all plugins?

Reply:
Hmm, that is a good question. I think it probably will. 😭

Reply:
I can try cloning this to make it available only to pulp_container.